### PR TITLE
chore(utils): replace keyCode usages with key for keyboard events

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1653)

### CODE REVIEW

**Review by atabel — APPROVED** [review]

File: src/utils/key-codes.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-export const ESC = 27;
-export const SPACE = 32;
-export const TAB = 9;
+export const LEFT = 'ArrowLeft';
+export const UP = 'ArrowUp';
+export const RIGHT = 'ArrowRight';
+export const DOWN = 'ArrowDown';
+export const ENTER = 'Enter';
+export const ESC = 'Escape';
+export const SPACE = ' ';
```
Comment (pladaria) [inline src/utils/key-codes.tsx:7]: :O

File: src/utils/key-codes.tsx
Comment (pladaria) [inline src/utils/key-codes.tsx:1]: I'd rename this file to "keys.tsx"

File: src/utils/key-codes.tsx (hunk 1 above)
Comment (pladaria) [inline src/utils/key-codes.tsx:7]: :O

**Review by pladaria — APPROVED** [review]

File: src/utils/key-codes.tsx
Comment (marcoskolodny) [reply src/utils/key-codes.tsx:1]: done :)

**Pull request merged**